app/qml/map/MMMapController.qml

```diff
@@ -895,6 +895,7 @@ Item {
       map: mapCanvas
       positionMarkerComponent: positionMarker
       recordingMapTool.centeredToGPS: root.centeredToGPS
+      centerToGPSOnStartup: __appSettings.autolockPosition
```

Comment:
Redundant information here, it is already set to the value of `__appSettings.autolockPosition` in `MMRecordingTools`.